wrappers/services_test.go

```diff
@@ -2973,6 +2977,72 @@ func (s *servicesTestSuite) TestQueryDisabledServicesActivatedServices(c *C) {
 	}
 }
 
+func (s *servicesTestSuite) TestQueryDisabledServicesUserServices(c *C) {
```

Comment:
do we want a test for with both user services and system services as well?

Reply:
Yea this is a great point, will add it in